server/grpc/server.go

```diff
@@ -106,11 +107,13 @@ func (s *Server) register() error {
 	coreServiceServer := NewServiceServer(s.sdk)
 	instanceServer := NewInstanceServer(s.sdk)
 	serviceServer := service.NewServer(s.sdk)
+	apiServer := api.NewServer(s.sdk)
```

Comment:
```suggestion
	apiExecutionServer := api.NewServer(s.sdk)
```
the api should be `api.NewExecutionServer(s.sdk)` in the first place so could be good to be more explicit on the variables here otherwise it's really confusing

Reply:
We'll change it :)